### PR TITLE
[BUGFIX] In the web layout module the checkbox "Show hidden content e…

### PR DESCRIPTION
**Pull request by monosize**
…lements" has no effect for nested flux grids.

### CODE REVIEW

File: Classes/View/PreviewView.php [hunk 1]
```diff
@@ -482,6 +482,7 @@ protected function getRecords(PageLayoutView $view, array $row, $area) {
 		// records based on them having remove placeholders.
 		$condition = "AND tx_flux_parent = '" . $row['uid'] . "' AND tx_flux_column = '" . $area . "' ";
 		$condition .= "AND colPos = '" . ContentService::COLPOS_FLUXCONTENT . "' ";
+		$condition .= $view->tt_contentConfig['showHidden'] ? '' : 'AND hidden = 0 ';
```
Comment (xf-) [inline Classes/View/PreviewView.php]: Please write a compare like TRUE ===
Also you can use a Elvis operator ?: 'AND hidden = 0 '

**Comment by xf-** [comment]
Commit message should be a little bit shorter. (you can use multiline)

**Comment by monosize** [comment]
A Compare with TRUE === and an Elvis operator ?:'AND hidden = 0' is not possible - the value from $GLOBAL['BE_USER']... is a string "0" or "1". Now the value is converted to an integer.

**Comment by xf-** [comment]
+1

**Pull request merged**